corehq/messaging/smsbackends/telerivet/models.py

```diff
@@ -99,8 +99,16 @@ def send(self, msg, *args, **kwargs):
             # These are account-related errors, retrying won't help
             msg.set_system_error(SMS.ERROR_TOO_MANY_UNSUCCESSFUL_ATTEMPTS)
         else:
+            try:
+                data = response.text
+            except Exception:
+                data = repr(response.content)
             raise TelerivetException(
-                "Received HTTP response status code %s from backend %s" % (response.status_code, self.pk)
+                "Received HTTP response status code %s (%s) from backend %s" % (
+                    response.status_code,
+                    self.pk,
+                    data,
```

Comment:
Did you intend to move `self.pk` from after "backend" to inside the parens? Or did you intend `data` to be inside the parens? ... Might be clearer if you switch to `.format()`.

Reply:
good catch. I will fix

Reply:
https://github.com/dimagi/commcare-hq/pull/24001/commits/93010dfb742013c17b3749f89b3617286209c3cf

Reply:
Eh, was reading too fast and missed your recommendation to `.format()` until after I had already pushed the commit. Not going to bother unless you object.

Reply:
Yeah, definitely don't bother. (Also, it's 2020 now: The days of `.format()` are over.)